specs/customizability.md

```diff
@@ -0,0 +1,350 @@
+- `validators` is a list of stakes of each genesis validator, given by the token collateral and their public key.
+- `accounts` is a list of genesis balances, given by tokens and FVM addresses.
+- `eam_permission_mode` defines who can deploy smart contracts, which could be anyone, or a set of whitelisted addresses.
+- `ipc` is only enabled if we want Fendermint to participate in an IPC hierarchy:
+    - `gateway` defines the parameters of the IPC `Gateway` actor:
+        - `subnet_id` defines the full `SubnetID` from the root
+        - `bottom_up_check_period` is the expected checkpoint frequency for the subnet. It is important that subnet nodes create checkpoints in the ledger at the same heights; doing otherwise would be a consensus failure.
+        - `majority_percentage` is the checkpoint quorum size expressed as a number between 0 and 100; it should be at least 67 to be Byzantine fault tolerant.
+        - `active_validators_limit` is the number of validators who can participate in the subnet consensus; it is important that both the parent and the child subnet agree on this value, so they select the same top validators the same way.
+
+The `genesis.json` file is usually constructed in one of two ways:
+
+- Using the `fendermint genesis` [CLI commands](https://github.com/consensus-shipyard/ipc/blob/specs/fendermint/app/options/src/genesis.rs) to incrementally add validators and accounts as described [here](https://github.com/consensus-shipyard/ipc/blob/specs/docs/fendermint/running.md#genesis).
+- Using the `fenderming genesis ipc from-parent` command to fetch the parameters from the subnet actor on the parent subnet and export them to file.
+
+# Gas policy
+
+The gas price in the FVM is by default [determined](https://github.com/filecoin-project/ref-fvm/blob/c39d880d086aa2e771c7190163436e02715d80f3/fvm/src/machine/mod.rs#L156) by the network version when the `NetworkConfig` is created, but could further be customised by assigning to the `price_list` field. Should Fendermint have to do that, it would be in the [constructor](https://github.com/consensus-shipyard/ipc/blob/7af25c4c860f5ab828e8177927a0f8b6b7a7cc74/fendermint/vm/interpreter/src/fvm/state/exec.rs#L132) of the `FvmExecState`.
+
+The `base_fee` set in genesis is also part of the gas policy. Currently it’s a static value, but it’s part of the upgradable parts of the `FvmExecState` and `FvmStateParams`, with its history maintained by the `App`. The interpreters could have logic to change its value based on the blocks they process.
+
+Also take a look at the following issue to see another proposal to use the `chainmetadata` actor to track the `base_fee` :
```

Comment:
Add TODO or link to the issue?

Reply:
The link is to an internal linear issue, will remove this and mention adding this as a GH issue